R/netcoupler_in.R

```diff
@@ -0,0 +1,123 @@
+
+# return: for each metabolite all possible models with different adjacency subsets and complete list of exposures and covariates
+
+net.coupler.in <- function(graph_skel, dat, dat_compl, exp_dat, DE) {
```

Comment:
As with the others, there were duplicates. See individual commit messages for more details.